make reviewable fails

### Describe the bug a clear and concise description of what the bug is.

```
INFO[2023-10-24T05:08:09-07:00] Generating README Documentation for chart charts/flux2-notification
Error: unknown flag: --helm3
Usage:
  unittest [flags] CHART [...]

Flags:
      --color                  enforce printing colored output even stdout is not a tty. Set to false to disable color
  -d, --debug                  enable debug logging
  -q, --failfast               direct quit testing, when a test is failed
  -f, --file stringArray       glob paths of test files location, default to tests/*_test.yaml (default [tests/*_test.yaml])
  -h, --help                   help for unittest
  -o, --output-file string     output-file the file where testresults are written in JUnit format, defaults no output is written to file
  -t, --output-type string     output-type the file-format where testresults are written in, accepted types are (JUnit, NUnit, XUnit, Sonar) (default "XUnit")
      --strict                 strict parse the testsuites
  -u, --update-snapshot        update the snapshot cached if needed, make sure you review the change before update
  -v, --values stringArray     absolute or glob paths of values files location to override helmchart values
  -s, --with-subchart charts   include tests of the subcharts within charts folder (default true)

unknown flag: --helm3
Error: plugin "unittest" exited with error
make: *** [unittests] Error 1
```

### What's your helm version?

v3.10

### What's your kubectl version?

1.28.x

### What's the chart version?

main branch

### What happened?

_No response_

### What you expected to happen?

_No response_

### How to reproduce it?

_No response_

### Enter the changed values of values.yaml?

_No response_

### Enter the command that you execute and failing/misfunctioning.

make reviewable

### Anything else we need to know?

_No response_

--- a/.github/workflows/lint-test.yaml
+++ b/.github/workflows/lint-test.yaml
@@ -35,7 +35,7 @@ jobs:
         if: steps.list-changed.outputs.changed == 'true'
         run: |
           helm env
-          helm plugin install https://github.com/quintush/helm-unittest.git --version 0.2.11
+          helm plugin install https://github.com/quintush/helm-unittest.git --version 0.3.5
 
       - name: Run chart-testing (lint)
         run: ct lint --config ct.yaml

--- a/CONTRIBUTING.md
+++ b/CONTRIBUTING.md
@@ -48,7 +48,7 @@ Once changes have been merged, the release job will automatically run to package
 
 * Make your changes
 * run: ```helm install --dry-run charts/flux2/ --generate-name```
-* run: ```helm unittest --helm3 --file 'tests/*.yaml' charts/flux2/```
+* run: ```helm unittest --file 'tests/*.yaml' charts/flux2/```
     * add ```-u``` if you need to update the compare snapshot in \_\_snapshots\_\_
 * bump chart version if necessary
 * run: ```make helmdocs```

--- a/Makefile
+++ b/Makefile
@@ -118,10 +118,10 @@ generate: fetch
 	@$(OK) Generating CRDs
 
 unittests:
-	@helm unittest --helm3 --file tests/*.yaml --file 'tests/**/*.yaml' charts/flux2-multi-tenancy/
-	@helm unittest --helm3 --file tests/*.yaml --file 'tests/**/*.yaml' charts/flux2/
-	@helm unittest --helm3 --file tests/*.yaml --file 'tests/**/*.yaml' charts/flux2-sync/
-	@helm unittest --helm3 --file tests/*.yaml --file 'tests/**/*.yaml' charts/flux2-notification/
+	@helm unittest --file tests/*.yaml --file 'tests/**/*.yaml' charts/flux2-multi-tenancy/
+	@helm unittest --file tests/*.yaml --file 'tests/**/*.yaml' charts/flux2/
+	@helm unittest --file tests/*.yaml --file 'tests/**/*.yaml' charts/flux2-sync/
+	@helm unittest --file tests/*.yaml --file 'tests/**/*.yaml' charts/flux2-notification/
 
 check-diff:
 	@$(INFO) checking that branch is clean

--- a/ct.yaml
+++ b/ct.yaml
@@ -3,4 +3,4 @@ remote: origin
 target-branch: main
 validate-maintainers: false
 additional-commands:
-  - helm unittest --helm3 --strict --file tests/*.yaml --file 'tests/**/*.yaml' {{ .Path }}
+  - helm unittest --strict --file tests/*.yaml --file 'tests/**/*.yaml' {{ .Path }}

